lib/explorers/api-security.explorer.ts

```diff
@@ -1,4 +1,8 @@
 import { DECORATORS } from '../constants';
+import {
+  SwaggerBearerAuthOption,
+  SwaggerSecurityDefinitionRules
+} from '../interfaces/swagger-bearer-auth-options.interface';
 
 export const exploreGlobalApiSecurityMetadata = metatype => {
```

Comment:
I'm not sure what `globalApiSecurity` refers to.

Reply:
I'm not entirely sure, but maybe globalApiSecurity refers to those securityDefinitions for all resources (via Authorize button at top of the page in SwaggerUI).

The other one, `exploreBearerSecurityOptions` might refer to other bearer than can be added on a particular endpoint (via Lock button at right-top of resource row in SwaggerUI.

I'm not entirely sure though, I'm new to this repository. Can someone confirm this?

Reply:
I think it refers to the securityDefinition for the whole Controller instead of individual Route Handler

/users // this is global security
|__/register
|__/login
|__/profile // this is lower level security

Reply:
Make sense. So, now, when we'll be using 
```
new DocumentBuilder()
      .addBearerAuth('x-refresh-token', 'header', 'apiKey')
      .addBearerAuth('x-authorization-token', 'header', 'apiKey')
```

We should have all these tokens as global security definitions.
Then we'll add to each protected endpoint
```

Reply:
After I added the logic for Controller level @ApiBearerAuth and tested, I can confirm that it's the right behavior to have the logic in the globalApiSecurityMetadata as well.